### PR TITLE
Update docker dependencies for Gulp

### PR DESCRIPTION
**Pull request by Artiume**
**Changes**

Performance improvement and various updates https://github.com/jellyfin/jellyfin-web/pull/862#issuecomment-593440878
**Issues**

### CODE REVIEW

**Comment by Artiume** [comment]
Verified amd64 and armhf. Did not verify arm64

**Comment by Artiume** [comment]
@joshuaboniface you want to give this a quick review? i think we should push it to fix nightlies.

File: Dockerfile [hunk 1]
```diff
@@ -3,7 +3,7 @@ ARG FFMPEG_VERSION=latest
 
 FROM node:alpine as web-builder
 ARG JELLYFIN_WEB_VERSION=master
-RUN apk add curl git \
+RUN apk add curl git zlib zlib-dev autoconf g++ make libpng-dev gifsicle alpine-sdk automake libtool make gcc musl-dev nasm \
```
Comment (dkanada) [inline Dockerfile]: Is yarn an included dependency with the alpine images?

Comment (Artiume) [reply Dockerfile]: I'm not sure

**Comment by Artiume** [comment]
https://github.com/jellyfin/jellyfin-web/pull/862#issuecomment-593440878

**Comment by JustAMan** [comment]
@Artiume I asked for adding a comment _inside_ the Dockerfiles, not here ;)

**Comment by Artiume** [comment]
🤦‍♂️

**Pull request merged**